mtools/mlaunch/mlaunch.py

```diff
@@ -266,7 +271,23 @@ def init(self):
         else:
             first_init = True
 
-        # check if authentication is enabled, make key file       
+        # Check if config replicaset is applicable to this version
+        if self.args['config_replicaset']:
+
+                binary="mongod"
+                if self.args and self.args['binarypath']:
+                    binary = os.path.join( self.args['binarypath'], binary)
+                ret = subprocess.Popen(['%s --version'%binary], stderr=subprocess.STDOUT, stdout=subprocess.PIPE, shell=True)
+                out, err = ret.communicate()
+
+                buf = StringIO(out)
+                current_version = buf.readline().rstrip('\n')[-5:]
+                print "current version is %s"%current_version
```

Comment:
this print statement should probably be removed. we don't want different output just because the `--config-replicaset` option was chosen.